emission/analysis/modelling/tour_model/representatives.py

```diff
@@ -0,0 +1,185 @@
+# standard imports
+import numpy
+import math
+import copy
+
+# our imports
+from emission.core.wrapper.trip import Trip, Coordinate
+
+"""
+This class creates a group of representatives for each cluster
+and defines the locations that the user visits from those clusters.
+
+The purpose of this class is to get the list of clusters with 
+start and end points to create the tour graph.
+
+To use this class, as input it takes
+- data: A list of trip objects
+- labels: A list of integers that define the clusters on the data.
+"""
+
+class representatives:
+
+    def __init__(self, data, labels):
+        self.data = data
+        if not self.data:
+            self.data = []
+        self.labels = labels

```

Comment:
the labels are a list of integers that define the clusters on the list of trips. If you run this from cluster pipeline, the labels are created there.